### ISSUE TITLE
Wrong semantics learned for "in front of"

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#719]
[slot1_in front of_slot2.pdf](https://github.com/isi-vista/adam/files/4381431/slot1_in.front.of_slot2.pdf)

This is probably related to #573 and/or #712

**Comment by LichMaster98** [comment]
The new integrated learner also is unable to learn this distinctly from "behind"

**Comment by gabbard** [comment]
@LichMaster98 : Was the previous learner able to do so?  And does it capture it but with the directions reverse? (that is, _in front of_ and _behind_ do have the same meaning except for their argument positons)

**Comment by LichMaster98** [comment]
@gabbard I don't believe the previous learner was able to distinguish between these two. I believe it's related to how we encode the `FacingAddresseeAxis` information -- The patterns end up identical from what I can tell. When we first introduce "A ball behind a box" the learner thinks it's saying "A ball in front of a box"
![image](https://user-images.githubusercontent.com/8626833/86853065-d2511780-c083-11ea-9f07-77e0bf75db6d.png)
